packages/docs/src/locales/en/en.json

```diff
@@ -1272,6 +1272,21 @@
       "events": {},
       "methods": {},
       "slots": {}
+    },
+    "VaSplit": {
+      "props": {
+        "vertical": "Changes `va-split` orientation to vertical.",
+        "minSize": "Minimal size of the each panel (%).",
```

Comment:
Do we able to make left minSize as 50%, but right is 16px?

Reply:
No in current state. It's possible to implement this but, honestly, i haven't seen any cases with panels different minimal size.

Reply:
Thanks, now i see.

I will implement this in current pull request scope.